android/java/res/layout/brave_toolbar.xml

```diff
@@ -27,33 +27,15 @@
           android:layout_gravity="center"
           android:contentDescription="@string/accessibility_toolbar_btn_brave_wallet" />
 
-      <org.chromium.chrome.browser.BraveBadge
-            android:id="@+id/brave_wallet_badge_small"
-            android:layout_width="16dp"
-            android:layout_height="16dp"
-            android:layout_marginStart="28dp"
-            android:layout_marginTop="6dp"
-            android:gravity="center"
-            android:visibility="invisible"
-            android:background="@drawable/brave_badge_shields_background_small"
-            android:textColor="@android:color/white"
-            android:translationZ="1dp"
-            android:fontFamily="sans-serif-bold"
-            android:textSize="12sp" />
-
-      <org.chromium.chrome.browser.BraveBadge
-            android:id="@+id/brave_wallet_badge_large"
-            android:layout_width="18dp"
-            android:layout_height="16dp"
-            android:layout_marginStart="28dp"
-            android:layout_marginTop="6dp"
-            android:gravity="center"
-            android:visibility="invisible"
-            android:background="@drawable/brave_badge_shields_background_large"
-            android:textColor="@android:color/white"
-            android:translationZ="1dp"
-            android:fontFamily="sans-serif-bold"
-            android:textSize="12sp" />
```

Comment:
nit: just curious, was there any issue with showing `brave_wallet_badge_small` as badge? as this can be used if we ever decides to show any count

Reply:
yeah, though just wanted to keep the more flexible approach :)